libs/packages/layouts/src/lib/feature/header/header.component.html

```diff
@@ -1,89 +1,54 @@
-<sds-top-banner *ngIf="showTopBanner" 
-[description]="topBannerDescription"
-></sds-top-banner>
-<div
-  class="usa-overlay"
-  [class.is-visible]="mobileNavActive"
-  (click)="closeMobileNav()"
-></div>
+<sds-top-banner *ngIf="showTopBanner" [description]="topBannerDescription"></sds-top-banner>
+
+<div *ngIf="alertsTemplate">
+  <ng-container [ngTemplateOutlet]="alertsTemplate"></ng-container>
+</div>
+
+<div class="usa-overlay" [class.is-visible]="mobileNavActive" (click)="closeMobileNav()"></div>
 <header class="usa-header usa-header--extended">
   <div class="usa-navbar">
-      <div *ngIf="showHeaderLogo; else fillerBlock" class="usa-logo" id="extended-logo">
-        <ng-container
-          [ngTemplateOutlet]="
+    <div *ngIf="showHeaderLogo; else fillerBlock" class="usa-logo" id="extended-logo">
+      <ng-container [ngTemplateOutlet]="
             navigationHelper.isLinkInternal(model.home)
               ? homeLinkRouteTemplate
               : homeLinkHREFTemplate
-          "
-          [ngTemplateOutletContext]="{ $implicit: model.home }"
-        ></ng-container>
-      </div>
+          " [ngTemplateOutletContext]="{ $implicit: model.home }"></ng-container>
+    </div>
     <button #usaNavOpen class="usa-menu-btn" (click)="openMobileNav()">
       <fa-layers [fixedWidth]="true" size="3x">
-        <fa-icon
-          [icon]="['fas', 'square']"
-          [classes]="['text-primary']"
-        ></fa-icon>
-        <fa-icon
-          [icon]="['sds', 'bars']"
-          [classes]="['icon-bars']"
-          transform="shrink-6"
-        ></fa-icon>
-        <fa-layers-counter
-          *ngIf="hasCounter()"
-          [classes]="['icon-layers-counter']"
-        ></fa-layers-counter>
+        <fa-icon [icon]="['fas', 'square']" [classes]="['text-primary']"></fa-icon>
```

Comment:
Relace fa-icon with sds-icon.

Reply:
Keeping for now as there is no sds icon replacement for square icon